snuba/datasets/processors/spans_processor.py

```diff
@@ -357,6 +355,7 @@ def process_message(
             "retention_days": retention_days,
             "partition": metadata.partition,
             "offset": metadata.offset,
+            "platform": "",
```

Comment:
The `common_span_fields` are used to hold fields which need to be pushed down to each individual span. I would avoid adding `platform` here.

Reply:
removed